SINGLE.md

```diff
@@ -0,0 +1,161 @@
+# Single Execution
+
+## Approving the transaction
+
+### 1. Update repo and move to the appropriate folder for this rehearsal task:
+
+```
+cd superchain-ops
+git pull
+just install
+```
```

Comment:
Step is called "Update repo and move to the appropriate folder for this rehearsal task" but there is no example `cd` command here